apps/sms/views/conversation/index.html

```diff
@@ -46,7 +46,6 @@
   <script defer src="/desktop-mock/navigator_moz_settings.js"></script>
   <script defer src="/desktop-mock/navigator_moz_mobilemessage.js"></script>
   <script defer src="/desktop-mock/navigator_moz_contacts.js"></script>
-  <script defer src="/desktop-mock/async_storage.js"></script>
```

Comment:
As we discussed offline - it's not easy to use mock async storage when drafts are used from window and shared worker - so disabling it for now.